docs/basics.adoc

```diff
@@ -504,6 +504,51 @@ Connecting to Gremlin-Server (127.0.0.1:8182)... OK (connected to 127.0.0.1:8182
 Run gremlin.sh to connect.
 ----
 
+==== Running JanusGraph Server as a daemon
+
+The JanusGraph https://github.com/JanusGraph/janusgraph/releases[release] also comes with an init script that can be found at `bin/gremlin-server.sh`. The `gremlin-server.sh` script was upgraded to an init script which supports start, stop, status, and restart operations and was introduced in versions 0.2.3, 0.3.2, and 0.4.0. It's usage is backwards compatible and can still be run in the foreground by invoking `./bin/gremlin-server.sh` with an option parameter of a gremlin server yaml file.
+
+If running JanusGraph as a daemon it is recommended to rename the pre-packaged to simply `janusgraph`, after unzipping, and move the distribution to `/opt/`.
+
+[source, text]
+----
+mv janusgraph-$VERSION-hadoop2 /opt/janusgraph
+----
+
+For System V you can symbolically link or copy `gremlin-server.sh` to `/etc/init.d/`. You will also need to copy `conf/janusgraph.env` to `/etc/default/janusgraph` and make sure that `$JANUSGRAPH_HOME` and `JANUSGRAPH_YAML` are pointing at the correct locations.
+
+**Note** Both of these examples assume that `JANUSGRAPH_HOME` is `/opt/janusgraph`
+
+Symbolic Link
+[source, text]
+----
+ln -s bin/gremlin-server.sh /etc/init.d/janusgraph
+cp conf/janusgraph.env /etc/default/janusgraph
```

Comment:
* symlink target should not use a relative path
* second line should use `ln -s`